x11-misc/ydotool/ydotool-1.0.4-r2.ebuild

```
@@ -9,7 +9,7 @@ DESCRIPTION="Generic command-line automation tool (no X!)"
 HOMEPAGE="https://github.com/ReimuNotMoe/ydotool"
 SRC_URI="https://github.com/ReimuNotMoe/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
-LICENSE="GPL-3"
+LICENSE="AGPL-3"
```

```suggestion
LICENSE="AGPL-3+"
```

The text in each file header says

>     This program is free software: you can redistribute it and/or modify
>    it under the terms of the GNU Affero General Public License as
>    published by the Free Software Foundation, either version 3 of the
>    License, **or (at your option) any later version.**

So we need to put that part into `LICENSE`.

https://devmanual.gentoo.org/general-concepts/licenses/index.html#gpl-x-vs-gpl-x